lib/std/os/linux/io_uring.zig

```diff
@@ -606,8 +654,44 @@ pub const IO_Uring = struct {
         return sqe;
     }
 
+    /// Queues (but does not submit) an SQE to perform an async zerocopy `send(2)`.
+    /// Returns a pointer to the SQE.
```

Comment:
Add some text to the `send_zc` comment about the required lifetime of `user_data`?  (I'm not entirely sure, but for a zero-copy send, the data needs to stay alive until the caller gets an ack back from the ring, right?)  Similarly for the other `_zc` functions.

Reply:
Great point, buffers must be alive until operation is fully finished.

Will add this:
> This operation will most likely produce two CQEs. The flags field of the
first cqe may likely contain IORING_CQE_F_MORE, which means that there will
be a second cqe with the user_data field set to the same value. The user
must not modify the data buffer until the notification is posted. The first
cqe follows the usual rules and so its res field will contain the number of
bytes sent or a negative error code. The notification's res field will be
set to zero and the flags field will contain IORING_CQE_F_NOTIF. The two
step model is needed because the kernel may hold on to buffers for a long
time, e.g. waiting for a TCP ACK. Notifications are responsible for controlling
the lifetime of the buffers. Even errored requests may generate a
notification.